src/clib/lib/enkf/model_config.cpp

```diff
@@ -444,6 +446,9 @@ void model_config_init(model_config_type *model_config,
         model_config_set_enspath(
             model_config,
             config_content_get_value_as_abspath(config, ENSPATH_KEY));
+    else
+        model_config_set_enspath(model_config,
+                                 fs::absolute(DEFAULT_ENSPATH).c_str());
```

Comment:
doesn't it make more sense to add the `fs::absolute` in line 262, in `model_config_alloc_empty`? that seems to be where the inconsitent value `storage` is written.

Reply:
Good point, initial solution required access to the `config_content` object until I realized I don't really need it :)